fendermint/docker/docker_entry.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# This script should be used as ENTRYPOINT for fendermint docker image.
+
+CMD=$1
+
+if [[ $CMD == 'ipc-cli' ]]; then
+  /usr/local/bin/ipc-cli "${@:2}"
```

Comment:
```suggestion
  exec ipc-cli "${@:2}"
```
So it handled signals.

Reply:
Yeah the `$@` is the array that contains all argument for this script.
`${@:2}` means its subarray starting from the second element.
The double quote is not compsiultory but recommended , see https://github.com/koalaman/shellcheck/wiki/SC2068.